_docs/getting_started/install.md

```diff
@@ -20,10 +20,12 @@ And if you don't &mdash; pick your operating system below:
 
 ### Step 1:
 
-Install the requirements using apt-get:
+Install the requirements using your package manager:
 
 ```
-$ sudo apt-get -y install nodejs-legacy npm
+debian-host$ sudo apt-get -y install nodejs-legacy npm
+
+fedora-host$ sudo yum install nodejs npm
 ```
```

Comment:
Can you split these in 2 please? Something like:

On Debian-based distributions:
```
sudo ...
```

On Fedora:
```
sudo ...
```